config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -48,8 +48,14 @@ presubmits:
     spec:
       # unit tests have no business requiring root or doing privileged operations
       securityContext:
-        runAsUser: 2000
+        # NOTE: these are arbitrary non-root values. They don't exist in the
+        # image and don't need to, the unit tests should only write to TMPDIR
+        runAsUser: 2001
+        runAsGroup: 2010
         allowPrivilegeEscalation: false
+        capabilities:
```

Comment:
inexplicably this is not showing up on the real prowjob https://prow.k8s.io/prowjob?prowjob=05d2ab37-db78-11eb-a74b-4e89bee5b562, only the runAsGroup

I think these are the correct keys and indentation ...?

Reply:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/92316/pull-kubernetes-unit-experimental/1411066357079871488/podinfo.json

Reply:
I think I know why,

this is the PodSecurityContext which does not have fields allowPrivilegeEscalation and capabilities. Those are in the containers SecurityContext

Reply:
You will need to add a securityContext to each of the containers. so that in the end it looks like
```
Pod
 spec:
   securityContext: # all containers in this pod will run with uid: 2001 and gid: 2010
     runAsUser: 2001
     runAsGroup: 2010
  containers:
  - name: blah
     securityContext:
       allowPrivilegeEscalation: false
       capabilities:
         drop: ["ALL"]
 ```

Reply:
This is why we need https://github.com/kubernetes/test-infra/pull/21075

Thanks

Reply:
https://github.com/kubernetes/test-infra/pull/22784